pages/tool_assembly/NO_TSD_assembly.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Norwegian ELIXIR tools assembly for sensitive data - TSD
+contributors: [Tina Visnovska, Federico Bianchini, Korbinian Bösl]
+summary: This is a Data Management tools assembly for sensitive data around TSD. TSD as an infrastructure is aimed for researchers in Norway and their collaborators, but can be used by anyone.
+tags: [plan, collect, analyse, preserve, share, reuse, IT support, data manager, researcher, human data, DMP, storage, sensitive]
```

Comment:
I am not sure that linking all assemblies everywhere would be an advantage. We could discuss a systematic approach in one of our meetings.

Reply:
Ok. also just noticed that there is unexpected behaviour on the problem pages from that. Better idea? I think we should at least cover the stages we cover?